### PR TITLE
Kotlin arrayOfNulls() term entry : Issue #5310

### LINKED ISSUE #5310: [Term Entry] Kotlin Arrays .arrayOfNulls()
### Reason/inspiration (optional)

We would like a new term entry on the `arrayOfNulls` under the 'Arrays` concept on Koltin. The entry should go in a new file under `content/kotlin/concepts/arrays/terms/arrayOfNulls/arrayOffNulls.md`.

The entry should include:

- An introduction to the term
- A `Syntax` section that provides the syntax for the concept (if applicable)
- An `Example` section that provides an example demonstrating the concept in use (if applicable)

Please refer to the [concept entry template](https://github.com/Codecademy/docs/blob/main/documentation/concept-entry-template.md), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md) and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue.

### Entry Type (select all that apply)

New Term for Existing Concept Entry

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [X] Labels added
- [ ] Issue is assigned

### PR DESCRIPTION
**Pull request by austinwdigital**
### Description

Added term entry for Kotlin arrayOfNulls(). Includes a description, the syntax and an example of its use.

### Issue Solved

Closes #5310 

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] I have linked any issues that are relevant to this PR in the `Issues Solved` section.

### CODE REVIEW

File: content/kotlin/concepts/arrays/terms/arrayOfNulls/arrayOfNulls.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+  - 'Android'
+  - 'Arrays'
+  - 'Map'
+  - 'Kotlin'
+CatalogContent:
+  - 'learn-kotlin'
+  - 'paths/computer-science'
+---
+
+In Kotlin, the **`arrayOfNulls()`** function creates an array of a specified size with all elements initialized as `null`. This method is useful when you need to create an array of a specific size but don't yet have the values to initialize it with. This allows you to assign values at a later point.
```
Comment (Radhika-okhade) [inline content/kotlin/concepts/arrays/terms/arrayOfNulls/arrayOfNulls.md:17]:
```suggestion
In Kotlin, the **`arrayOfNulls()`** function creates an array of a specified size with all elements initialized as `null`. This method is useful for creating an array with a defined size when the initial values are not yet available, allowing for value assignment at a later point.
```

File: content/kotlin/concepts/arrays/terms/arrayOfNulls/arrayOfNulls.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+## Syntax
+
+```pseudo
+fun <T> arrayOfNulls(size: Int): Array<T?>
+```
+
+- `T`: The type of the elements in the array.
```
Comment (Radhika-okhade) [inline content/kotlin/concepts/arrays/terms/arrayOfNulls/arrayOfNulls.md:25]:
```suggestion
- `T`: Represents the type of elements in the array.
```

File: content/kotlin/concepts/arrays/terms/arrayOfNulls/arrayOfNulls.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+- `size`: An integer specifying the size of the array to create.
+
+It returns an `Array<T?>` of the specified size, initialized with nulls.
```
Comment (Radhika-okhade) [inline content/kotlin/concepts/arrays/terms/arrayOfNulls/arrayOfNulls.md:28]:
```suggestion
It returns an `Array<T?>` of the specified size, initialized with `null`.
```

File: content/kotlin/concepts/arrays/terms/arrayOfNulls/arrayOfNulls.md [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+    // Assign values to some elements in the array
+    nullArray[0] = 2
+    nullArray[1] = 4
+
+    // Print the array after assigning values
+    println(nullArray.contentToString())
+}
+```
+
+The code above generates the following output:
```
Comment (Radhika-okhade) [inline content/kotlin/concepts/arrays/terms/arrayOfNulls/arrayOfNulls.md:48]:
```suggestion
This example results in the following output:
```

File: content/kotlin/concepts/arrays/terms/arrayOfNulls/arrayOfNulls.md [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+
+    // Assign values to some elements in the array
+    nullArray[0] = 2
+    nullArray[1] = 4
+
+    // Print the array after assigning values
+    println(nullArray.contentToString())
+}
+```
+
```
Comment (Radhika-okhade) [inline content/kotlin/concepts/arrays/terms/arrayOfNulls/arrayOfNulls.md:47]:
```suggestion
## Output
```

File: content/kotlin/concepts/arrays/terms/arrayOfNulls/arrayOfNulls.md [hunk 6]
```diff
@@ -0,0 +1,52 @@
...
+This example results in the following output:
+
```
Comment (Radhika-okhade) [inline content/kotlin/concepts/arrays/terms/arrayOfNulls/arrayOfNulls.md:49]: Give line break before and after output block
```suggestion
## Output
```

**Comment by Radhika-okhade** [comment]
Hey! @austinwdigital  Thank you for contributing to Codecademy docs. 🙂
The doc is very well written. 🚀 
It's good to go for a second review.

**Review by Radhika-okhade — APPROVED** [review]

**Review by mamtawardhani — APPROVED** [review]
Thank you for contributing to Codecademy Docs @austinwdigital 😄 

The entry looks good to be merged! 🚀

**Comment by austinwdigital** [comment]
Thank you, @mamtawardhani and @Radhika-okhade, I appreciate it! 🙂

**Pull request merged**